src/Controller/SuggestionProviderController.php

```diff
@@ -116,15 +119,6 @@ public function listAction(
     }
 
     /**
-     * @param SuggestionProviderInterface $matcher
-     * @param string                      $group
-     */
-    public function addSuggestionProvider(SuggestionProviderInterface $matcher, $group)
```

Comment:
this is a BC break if somebody relied on adding providers later, e.g. in a compilar pass

Reply:
mhh, got a hint from sensio guard not to use other public methods then actions in controllers, what to do? Changelog enough?

Reply:
ahh you wrote it :-)

Reply:
Added a BC-BREAK notice in change log, i think we can live ẃith that on an non major release.